egs/aishell/s10/chain/sgd_mc.py

```diff
@@ -0,0 +1,127 @@
+import torch
+from torch.optim.optimizer import Optimizer, required
+
+
+class SGD_MC(Optimizer):
```

Comment:
I think SgdMaxChange might be clearer?  and called sgd_max_change.py?  probably closer to google style guide.
Please make sure the added parameters are documented.